component/src/main/java/com/celements/photo/container/ImageUrlDim.java

```diff
@@ -0,0 +1,35 @@
+package com.celements.photo.container;
+
+public class ImageUrlDim {
+
+  private String url;
+  private String width = "";
+  private String height = "";
+
+  public ImageUrlDim(String url, int width, int height) {
+    this.url = url;
+    if (width > 0) {
```

Comment:
Please move the conversion to the getter. Do as less logic as possible in the constructor.
Moreover if the arguments are out of specification (e.g. negativ) throw an checked IlligalArgumentException. Fast failure helps finding bugs early.